src/inject/dynamic-theme/style-manager.ts

```diff
@@ -145,6 +148,7 @@ export function manageStyle(element: StyleElement, {update, loadingStart, loadin
 
     let isLoadingRules = false;
     let wasLoadingError = false;
+    let loadingLinkID = 0;
```

Comment:
Let's call it counter instead of ID. It becomes ID when you pass it into a function where the reference to the counter is lost.